src/Sarif.UnitTests/SarifCurrentToVersionOneTests.cs

```diff
@@ -1108,9 +1108,7 @@ public void SarifTransformerTests_ToVersionOne_ResultLocations()
                 ""region"": {
                   ""startLine"": 20,
                   ""startColumn"": 69,
-                  ""endColumn"": 74,
-                  ""offset"": 720,
-                  ""length"": 5
+                  ""endColumn"": 74
```

Comment:
The fact that the "v2" log above includes `offset` and `length`, rather than `byteOffset` and `byteLength`, explains why this v1 location is missing its `offset` and `length` properties. When you fix the v2 log above, the missing properties should reappear. It's worth asking why you're not seeing an exception when you deserialize the incorrect v2 log.

Reply:
Fixed